### ISSUE TITLE
Capability to get probabilities for each tag in crf_decode

### ISSUE THREAD

**Issue by agarwalishan** [tensorflow/addons#2088]
**Describe the feature and the current behavior/state.**
I am talking about this specific repository.
[https://www.tensorflow.org/addons/api_docs/python/tfa/text/crf_decode]()
For now crf_decode function returns the tag indices with highest probability and a score for the whole sequence.
But the crf_decode do not returns the probabilities for every tag for each token in sequence like we get in softmax.
So I request to please add this capability because probabilities of tags help us know the confidence score and many more things.

**Relevant information**
- Are you willing to contribute it (yes/no): yes
- Are you willing to maintain it going forward? (yes/no): no
- Is there a relevant academic paper? (if so, where): no
- Is there already an implementation in another framework? (if so, where): I guess in pytorch
- Was it part of tf.contrib? (if so, where): yes, tf.contrib.crf.crf_decode

**Which API type would this fall under (layer, metric, optimizer, etc.)**
It will fall under 'text'

**Who will benefit with this feature?**
I believe many developers around the globe are looking for this feature.

**Any other info.**
Confidence scores for any model are very important for further decisions and modifications.

**Comment by Ghostvv** [comment]
Hi, we implemented the probability for each tag here: https://github.com/RasaHQ/rasa/blob/master/rasa/utils/tensorflow/crf.py

**Comment by howl-anderson** [comment]
Regarding to `returns the probabilities for every tag for each token in sequence`.  Do you means you want the emission probability of each words?

if so, I think you can use `tfa.layers.CRF` to get the potentials (as logits).

In following sample code:

```python
layer = tfa.layers.CRF(4, use_boundary=False)
decoded_sequence, potentials, sequence_length, chain_kernel = layer(inputs)
```

`potentials` is the thing you want.

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
